survey-runner-routing/vpc_endpoint.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_vpc_endpoint" "private-dynamodb" {
+  vpc_id       = "${var.vpc_id}"
+  service_name = "com.amazonaws.${data.aws_region.current.name}.dynamodb"
+  route_table_ids = ["${aws_route_table.private.*.id}"]
+}
```

Comment:
Any value adding a policy to this to control the access to this?

Reply:
We explicitly have allow each task its permission to access DynamoDB, So if we added a policy here it would just be a combination of the permissions given already.
We will add a restriction to DynamoDB to only allow access from within the VPC but this is on a different repo